src/clj/fluree/db/query/exec/select.cljc

```diff
@@ -157,7 +177,14 @@
                          where/get-iri)
                      subj)]
       (json-ld-resp/format-node ds iri context compact spec iri-cache
-                                fuel-tracker error-ch))))
+                                fuel-tracker error-ch)))
+  (solution-value
+    [_ error-ch _]
+    (async/put!
+     error-ch
+     (ex-info "Graph crawl select statements cannot be used in subqueries."
```

Comment:
Another thing we should detect during the parsing phase.

Reply:
Note previous related comment

Reply:
Addressed in d99b491bb69a12e2c0fd175d737005e4ca2419b9